.gitignore

```diff
@@ -10,5 +10,7 @@ node_modules/
 
 ### IntelliJ IDEA ###
 .idea
-### VS Code ###
-.vscode
+
+### jasmine-reporters ###
+!/spec/support/jasmine.js
```

Comment:
Why is line 15 needed?

Reply:
> Why is line 15 needed?

Because all `*.js` files is set to be ignored on [line 2](https://github.com/parasoft/parasoft-findings-vsts/blob/b6e6e726ceeaf7cc55de6a42361c5924b7bb27d2/.gitignore#LL2C1-L2C1), but we need to include this `jasmine.js` file.